python/py-wntr/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        USEPA WNTR 0.2.1
+
+name                py-wntr
+
+categories-append   science
+platforms           darwin
+license             BSD
+maintainers         {gmail.com:jjstickel @jjstickel} openmaintainer
+
+description         The Water Network Tool for Resilience (WNTR)
+long_description    The Water Network Tool for Resilience (WNTR) is an EPANET \
+                    compatible Python package designed to simulate and analyze \
+                    resilience of water distribution networks.
+
+checksums           rmd160  ae7313d03c8af77c59cb93cb6e0f18fa625650b2 \
+                    sha256  e9e3f7414055aa51011e70063f63042d283d6e0cc352815839ff0d43f97842ca \
+                    size    5878959
+
+python.versions     35 36 37
```

Comment:
Please consider not having py35 and py36 sub-ports.

Reply:
I set these to what upstream supports. I'm still on 3.6 myself.

Reply:
Well you are the maintainer so you are also supporting them.